src/main/java/uk/gov/digital/ho/hocs/casework/api/StageService.java

```diff
@@ -529,7 +549,9 @@ private void updateStages(Set<StageWithCaseData> stages) {
         for (StageWithCaseData stage : stages) {
             stagePriorityCalculator.updatePriority(stage, stage.getCaseDataType());
             daysElapsedCalculator.updateDaysElapsed(stage.getData(), stage.getCaseDataType());
-            stage.setTag(stageTagsDecorator.decorateTags(stage.getData(), stage.getStageType()));
+
+            //TODO: Remove after Workflow Implementation released
```

Comment:
We should raise a separate ticket and reference the ticket in this if they are going separately

Reply:
This has been added